### PR TITLE
Review

### PR DESCRIPTION
**Pull request by lqdung-lockon**

### CODE REVIEW

File: src/Eccube/Resource/template/admin/Setting/System/member.twig [hunk 1]
```diff
@@ -19,97 +19,142 @@ You should have received a copy of the GNU General Public License
...
+{% form_theme form 'Form/bootstrap_4_layout.html.twig' %}
 
 {% block javascript %}
-    <script>
-        function changeAction(action) {
-            document.form1.action = action;
-        }
-    </script>
+<script>
+function changeAction(action) {
```
Comment (lqdung-lockon) [inline src/Eccube/Resource/template/admin/Setting/System/member.twig:33]: don't use, please remove it

File: src/Eccube/Resource/template/admin/Setting/System/member.twig [hunk 2]
```diff
@@ -19,97 +19,142 @@ You should have received a copy of the GNU General Public License
...
                                                     {% else %}
-                                                        <li>
-                                                            <a href="{{ url('admin_setting_system_member_delete', {id: Member.id}) }}" {{ csrf_token_for_anchor() }} data-method="delete">{{'admin.setting.system.member.693'|trans}}</a>
-                                                        </li>
-                                                    {% endif %}
-                                                    {% if loop.first == false %}
-                                                        <li>
-                                                            <a href="{{ url('admin_setting_system_member_up', {id: Member.id}) }}"  {{ csrf_token_for_anchor() }} data-method="put" data-confirm="false">{{'admin.setting.system.member.694'|trans}}</a>
-                                                        </li>
+                                                        <a class="btn btn-ec-actionIcon action-delete"
```
Comment (lqdung-lockon) [inline src/Eccube/Resource/template/admin/Setting/System/member.twig:129]: Please change the alert style to the modal style.

File: src/Eccube/Resource/template/admin/Setting/System/member.twig [hunk 3]
```diff
@@ -19,97 +19,142 @@ You should have received a copy of the GNU General Public License
...
+                                                    <a class="btn btn-ec-actionIcon action-edit"
+                                                       href="{{ url('admin_setting_system_member_edit', { 'id' : Member.id }) }}"
+                                                       data-toggle="tooltip"
+                                                       data-placement="top"
+                                                       data-original-title="{{'admin.setting.system.member.691'|trans}}">
+                                                        <i class="fa fa-pencil fa-lg text-secondary" aria-hidden="true"></i>
+                                                    </a>
+                                                </div>
+                                                <div class="col-auto text-center">
+                                                    <a class="btn btn-ec-actionIcon action-up {% if loop.first %} disabled {% endif %}"
```
Comment (lqdung-lockon) [inline src/Eccube/Resource/template/admin/Setting/System/member.twig:98]: action up & down: please change to ajax handle.

**Comment by hoand-vn** [comment]
fixed

**Pull request closed without merging**